src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -457,8 +456,7 @@ private static synchronized void restartLobbyWatcher(final SetupPanelModel setup
 
   public static void resetLobbyHostOldExtensionProperties() {
     for (final String property : getProperties()) {
-      if (GameRunner.LOBBY_HOST.equals(property) || LobbyServer.TRIPLEA_LOBBY_PORT_PROPERTY.equals(property)
-          || GameRunner.LOBBY_GAME_HOSTED_BY.equals(property)) {
+      if (GameRunner.LOBBY_HOST.equals(property) || GameRunner.LOBBY_GAME_HOSTED_BY.equals(property)) {
```

Comment:
Just asking because I'm not sure...  Why was `TRIPLEA_LOBBY_PORT_PROPERTY` removed here instead of using the updated reference in `GameRunner`?

If removing it was intentional, might want to update the comment on the next line that refers to "3" properties.